src/_constants.scss

```diff
@@ -1,3 +1,4 @@
 // durations
-$duration-s: .2s;
-$duration-m: .5s;
+$duration-s: .1s;
+$duration-m: .25s;
+$duration-l: .5s;
```

Comment:
Not sure how I feel about `s, m, l` - that's what we use for sizing. @akdetrick - any thoughts?